LeadKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name            = "LeadKit"
-  s.version         = "0.3.2"
+  s.version         = "0.3.3"
```

Comment:
https://github.com/TouchInstinct/Podspecs/tree/master/LeadKit last version is 0.3.6, not 0.3.2 :(